test/cql-pytest/README.md

```diff
@@ -202,3 +206,71 @@ and practices be followed when writing new tests:
    changing those tests without good reason, and don't add new tests to
    any file in it. Put new tests in the `cql-pytest` directory, in any
    place except the `cassandra_tests` subdirectory.
+
+# Installing Cassandra
+
+As explained above, the ability to run cql-pytest tests against Cassandra
+makes it easier to write correct tests, to ensure compatibility with
+Cassandra, and sometimes to write tests for new Cassandra-inspired features
+before developing the feature in Scylla (this is so-called "test-driven
+development"). Unfortunately, in recent years modern Linux distributions
+dropped their "cassandra" package, so to run Cassandra you'll need to
+install it manually, and this section explains how. It's very easy, and
+don't worry - you don't even need to learn how to run Cassandra, as the
+"test/cql-pytest/run-cassandra" tool will do it for you.
+
+To be able to run Cassandra, you'll need either Java 8 or 11 installed on
+your system - Cassandra does not support more recent versions of Java.
+However, this old Java only needs to be installed *alongside* your favorite
+version of Java - it does not need to be the default Java on your system.
+The "run-cassandra" script will automatically pick the right version of
+Java from multiple versions installed on your system. Of modern Fedora,
+installing Java 11 as a secondary Java is as simple as `dnf install java-11`.
+
+## Precompiled Cassandra
+The easiest way to get Cassandra is to get a pre-compiled tar.
+Go to [Cassandra's download page](https://cassandra.apache.org/_/download.html)
+and pick the specific version you want to run, and download the `bin.tar.gz`
+file. For example, [4.1.4](https://dlcdn.apache.org/cassandra/4.1.4/apache-cassandra-4.1.4-bin.tar.gz).
+
+Open this tar in any directory you choose (you don't need to install it in
+any specific place), using `tar zxvf apache-cassandra-4.1.4-bin.tar.gz`.
+
+That's it! In the newly opened directory, you have bin/cassandra (as well
+as bin/nodetool and other things), which you will ask run-cassandra to use:
+
+```
+export CASSANDRA=/tmp/apache-cassandra-4.1.4/bin/cassandra
+test/cql-pytest/run-cassandra testfile.py::testfunc
+```
+
+## Building Cassandra from source code
+Usually, installing a pre-compiled Cassandra is enough. But in some cases
+you might want to test some unofficial or modified version of Cassandra,
+built from source. This is also not difficult:
+
+First, download the Cassandra source code, e.g., from github:
```

Comment:
```suggestion
First, download the Cassandra source code, e.g. from github:
```

Reply:
According to the manuals of style I read in the past, "e.g." must be followed with a comma so I always do. But I agree that in this case, the reasoning - even if you know Latin - is dubious. I'll remove it.